client/data/index.js

```diff
@@ -0,0 +1,19 @@
+//TODO, these should come from an endpoint/ data store.
```

Comment:
This is a placeholder for now to fetch data. It will be done separately in https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1718